src/pages/contact.js

```diff
@@ -1,5 +1,29 @@
+import { ImageWrapper } from '../components/image-wrapper/image-wrapper';
+import { FaLinkedinIn } from "react-icons/fa";
+import { FaGithub } from "react-icons/fa";
+import { FaTwitter } from "react-icons/fa";
+import { FaInstagram } from "react-icons/fa";
 export const ContactPage = () => {
     return <div>
-        Cotact me
-        </div>
+        <h3>Contact me </h3>
+        <p>Hello,</p>
+        <p>I am a web developer</p>
+        <p>If you want to do a project with me</p>
+        <p>Let's keep in touch</p>
+        <p>+90 542 323 92 48</p>
```

Comment:
No phone number online😉 and no email online

If someone would like to reach you they can use the platforms below